sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -903,6 +904,28 @@ def test_non_euclidean_kneighbors():
                   radius, metric='euclidean')
 
 
+def test_kneighbors_parallel():
+    X = iris.data
+    y = iris.target
+    X_train, X_test, y_train, y_test = train_test_split(X, y)
+    for algorithm in ALGORITHMS:
+        clf = neighbors.KNeighborsClassifier(n_neighbors=3,
+                                             algorithm=algorithm)
+        clf.fit(X_train, y_train)
+        y_1 = clf.predict(X_test)
+        dist_1, ind_1 = clf.kneighbors(X_test)
+        A_1 = clf.kneighbors_graph(X_test, mode='distance').todense()
+        for n_jobs in [2]:
```

Comment:
It's done elsewhere in testing. I don't know if joblib ignores it, but in any case this sort of load should be fine for the operating system to just switch on one core.